docs/source/conf.py

```diff
@@ -16,6 +16,77 @@ class AlgoKitRenderer(MystRenderer):
 
     """
 
+
+    @te.override
+    def render_class(self, item: ItemData) -> t.Iterable[str]:
+        """Create the content for a class."""
+        short_name = item["full_name"].split(".")[-1]
+        constructor = self.get_item(f"{item['full_name']}.__init__")
+        sig = short_name
+        if constructor and "args" in constructor:
+            args = self.format_args(
+                constructor["args"], self.show_annotations(item), ignore_self="self"
+            )
+            sig += f"({args})"
+
+        # note, here we can cannot yield by line,
+        # because we need to look ahead to know the length of the backticks
+
+        lines: list[str] = [f":canonical: {item['full_name']}"]
+        if self.no_index(item):
+            lines += [":noindex:"]
+        lines += [""]
+
+        # TODO overloads
+
+        if item.get("bases") and self.show_class_inheritance(item):
+            lines += [
+                "Bases: "
+                + ", ".join(
+                    [self._reformat_cls_base_myst(b) for b in item.get("bases", [])]
+                ),
+                "",
+            ]
+
+        if self.show_docstring(item):
+            lines.append(f"```{{autodoc2-docstring}} {item['full_name']}")
+            if parser_name := self.get_doc_parser(item["full_name"]):
+                lines.append(f":parser: {parser_name}")
+            lines.append("```")
+            lines.append("")
+
+            if self.config.class_docstring == "merge":
+                init_item = self.get_item(f"{item['full_name']}.__init__")
+                if init_item:
+                    lines.extend(
+                        [
+                            "Initialization",
```

Comment:
@daniel-makerx any idea on how section headings can be tweaked so that init section is collapsed under h4 instead h3 tag. The line that is od

This line initially was "```{rubric} Initialization```" - which is what renders the ### Initialization, removing the myst block and just keeping #### Initialization doesn't work either and instead it strips the h4 to h3 in the final md. 

This addresses the @larkiny 's request to remove init section from same level as the class description itself however setting h4 isn't trivial either. @daniel-makerx unless there is a simple way to tweak that without overriding the method in custom renderer - let me know if we are ok with with init section not being rendered as a section